src/Nowruz/modules/Jobs/modules/savedJobListing/useSavedJobListing.tsx

```diff
@@ -58,5 +61,23 @@ export const useSavedJobListing = () => {
     localStorage.setItem('source', 'saved');
   }, [page]);
 
-  return { page, setPage, jobsList, total: totalCount, PER_PAGE, isMobile, skillList, loading, loadPage };
+  useEffect(() => {
+    if (isMobile && page > 1) executeScroll();
```

Comment:
FMI: 1. in other `useEffect` you didn't use this condition, does it matter here?
2. can we not remove `useEffect` and call `executeScoll()` when we set the state of `jobsList`?

Reply:
1. changed it, at first I just focused on mobile view (since the ticket has mentioned just mobile view), but later I changed it also for desktop view
2. Actually no, if we executeScoll() immediately after setState, the ref would be null